charts/op-secrets/templates/waitjob.yaml

```
@@ -49,7 +49,7 @@ spec:
       restartPolicy: Never
       containers:
       - name: wait-op-secrets
-        image: bitnami/kubectl:latest
+        image: bitnamilegacy/kubectl:latest
```

Let's use `bitnamisecure`?
```suggestion
        image: bitnamisecure/kubectl:latest
```